extensions/amp-video-docking/0.1/amp-video-docking.js

```diff
@@ -1213,27 +1213,40 @@ export class VideoDocking {
 
   /**
    * @param {!../../../src/video-interface.VideoOrBaseElementDef} video
+   * @private
    */
   setPosterImage_(video) {
-    const attr = 'poster';
-
-    const {element} = video;
-
     const placeholderPoster = this.getPlaceholderRefs_()['poster'];
+    const posterSrc = this.getPosterImageSrc_(video.element);
 
-    if (!element.hasAttribute('poster')) {
+    if (!posterSrc) {
       toggle(placeholderPoster, false);
       return;
     }
 
-    const posterSrc = element.getAttribute(attr);
-
     toggle(placeholderPoster, true);
     setStyles(placeholderPoster, {
       'background-image': `url(${posterSrc})`,
     });
   }
 
+  /**
+   * @param {!Element} element
+   * @return {string|undefined}
+   * @private
+   */
+  getPosterImageSrc_(element) {
+    const poster = 'poster';
+    if (element.hasAttribute(poster)) {
+      return element.getAttribute(poster);
+    }
+    const imgEl = element.querySelector(
```

Comment:
Is `img[placeholder]` possible?

`amp-img[placeholder], [placeholder] amp-img` may cover more cases in case they wrap it in a div for some reason.

Reply:
[Anything is possible 💫](https://github.com/ampproject/amphtml/blob/master/extensions/amp-youtube/0.1/amp-youtube.js#L431).

Added `[placeholder] img`.

Reply:
Lol, there used to be a TODO there to switch to amp-img, oh well. :)